helm/docs/expose-teams-api.md

```diff
@@ -11,23 +11,44 @@
 
 # Exposing the `teams-api` Service
 
-You may wish to expose your FiftyOne Teams API for SDK access.
+There are two methods for SDK access to Fiftyone Teams
 
-You may expose your `teams-api` service in any manner that suits your deployment strategy.
-The following is one solution, but does not represent the entirety of possible solutions.
-Any solution allowing the FiftyOne Teams SDK to use websockets to access the `teams-api` service on port 80 should work.
+- Direct MongoDB connection
+- FiftyOne Teams API
 
-**NOTE**: The `teams-api` service uses websockets to maintain connections and allow for long-running processes to complete.
-Please ensure your Infrastructure supports websockets before attempting to expose the `teams-api` service.
-(e.g. You will have to migrate from AWS Classic Load Balancers to AWS Application Load Balancers to provide websockets support.)
+The database direct connection requires each user to have root database privileges.
 
-**NOTE**: If you are using file-based storage credentials, or setting environment variables, the same credentials must be shared with the `fiftyone-app` and `teams-api` pods.
-Voxel51 recommends the use of Database Cloud Storage Credentials, which can be configured at `/settings/cloud_storage_credentials`.
+The FiftyOne Teams API provides Role Based Access Control (RBAC) permissions.
+By default, the API is not exposed.
+To expose the FiftyOne Teams API, configure a
+Kubernetes Ingress to route traffic to the Kubernetes
+`teams-api` service on port 80 via the WebSocket protocol.
 
-## Adding a Second Host to the Ingress Controller (Host-Based Routing)
+We use WebSockets to maintain connections and enable long-running process execution.
+Before exposing the `teams-api` service,
+validate that your infrastructure supports the WebSockets protocol.
+(For example, you may need to replace AWS Classic Load Balancers (LB)
+with AWS Application Load Balancers (ALB) for WebSocket support.)
 
-1. Set `apiSettings.dnsName` to the hostname to route API requests to
-   (e.g. demo-api.fiftyone.ai)
+When using file-based storage credentials or setting environment variables, the
+same credentials must be shared with the `fiftyone-app` and `teams-api` pods.
+We recommend using Database Cloud Storage Credentials configured at
+`https://<DEPOY_URL>/settings/cloud_storage_credentials`.
```

Comment:
I'm not clear what this has to do with exposing the API...

Reply:
/shrug

https://github.com/voxel51/fiftyone-teams-app-deploy/commit/10fec67b04f914d3ac423b6abb167abb87e728ef sez that you added this to the section?  Maybe four months ago topher would be able to explain it?

Reply:
I removed this as suggested.